vendor-overwrites/colorpicker/colorconverter.js

```diff
@@ -74,16 +107,21 @@ const colorConverter = (() => {
     const first = num[0];
     if (/rgb/i.test(type)) {
       const k = first.endsWith('%') ? 2.55 : 1;
-      const [r, g, b] = num.map(s => parseFloat(s) * k);
-      return {type: 'rgb', r, g, b, a};
+      const [r, g, b] = num.map(s => {
+        const val = nonDigit.test(s.replace('%', '')) ? -1 : parseFloat(s) * k;
+        // Round the values when converting % to byte value or validation fails
+        return k === 1 ? val : Math.round(val);
+      });
+      return validateRGB(r, g, b) && validateAlpha(a) ? {type: 'rgb', r, g, b, a} : null;
     } else {
-      let h = parseFloat(first);
+      let h = nonDigit.test(first.replace('.', '').replace(/(deg|grad|rad|turn)/, '')) ? -1 : parseFloat(first);
```

Comment:
There should be no need to change the parsing process:
```js
if (!validHue(first)) {
  return null;
}
```

Reply:
This was the same issue as with rgb... entering alphabetic characters would be ignored once `parseFloat` was applied.